atf_eregs/templates/atf_eregs/poster_wrapper.html

```diff
@@ -18,6 +18,7 @@
     </p>
   </div>
 
+  <a href="#" class="clipboard-link">Copy to clipboard</a>
   <div class="poster extract">
```

Comment:
Let's go w/ progressive enhancement over including extra markup. To me that means adding a "clipboard-link" to the `div` and inserting the link via JS